api/controllers/UserController.js

```diff
@@ -283,7 +283,7 @@ module.exports = {
         user: req.user.name,
         type: "banUser",
         content: "Banned /u/" + req.params.username
-      });
+      }).exec(function () {});
```

Comment:
This, I guess, is a bug fix for 2.2, so add this as a separate PR to 2.2, and we'll get that in.